netbox/project-static/src/search.ts

```diff
@@ -11,12 +11,42 @@ function quickSearchEventHandler(event: Event): void {
   if (isTruthy(clearbtn)) {
     if (quicksearch.value === "") {
       clearbtn.classList.add("invisible");
+      clearLinkParams();
     } else {
       clearbtn.classList.remove("invisible");
     }
   }
 }
 
+/**
+ * Clear the existing search parameters in the link to export Current View.
+ */
+function clearLinkParams(): void {
+  const link = document.getElementById("current_view") as HTMLLinkElement;
```

Comment:
Could we give this element a slightly more descriptive name, to help prevent confusion?
```suggestion
  const link = document.getElementById("export_current_view") as HTMLLinkElement;
```